cves/CVE-2016-7122.yml

```diff
@@ -64,7 +68,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Function was vulnerable to an infinite loop due to the improper management
```

Comment:
I think a quick description of why an infinite loops is bad would be very useful

Reply:
added a sentence stating why infinite loops are bad